### ISSUE TITLE
Closures might issue false positive on Credo.Check.Readability.LargeNumbers

### ISSUE THREAD

**Issue by mememori** [rrrene/credo#354]
With the following input
```elixir
foo.(12_345)
```

The check will raise an warning while
```elixir
foo(12_345)
```
will not.

This is because the first will return the token `{:number, {1, 7, 13}, 12345}` while the second returns `{:number, {1, 5, 11}, 12345}` (note that simply adding a dot will change the column number by 2 ? Not sure why tho)

Elixir is 1.4.2, Credo is 0.7

**Comment by mememori** [comment]
Also note that this won't happen for every input number.

Happens on
`XX_XXX` and `XXX_XXX` but not on `X_XXX`. Didn't test with higher numbers. Obviously this is caused by the column position returned by the token.

This might be an issue on `elixir_tokenizer` since it is the one that is returning the token with that 2 values jump on column position

**Comment by rrrene** [comment]
@mememori Thx for reporting!

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍
